atlasdb-config/build.gradle

```diff
@@ -12,7 +12,10 @@ dependencies {
     compile('com.netflix.feign:feign-jaxrs:8.6.1') {
         exclude module: 'jsr311-api'
     }
-    compile "com.netflix.feign:feign-okhttp:8.6.1"
+    // versions below 8.10.0 have a bug where POST requets must have a body
+    compile('com.netflix.feign:feign-okhttp:8.10.1') {
```

Comment:
I gave that a try before writing this PR and it causes other dependencies to get out of wack.  I figure we can do that upgrade later, but this fixes clustered services and can be pushed ahead now?